frontend/src/nc/css/App.css

```diff
@@ -406,7 +406,7 @@ html[data-w-dynpage] [data-w-cloak] {
   cursor: pointer;
   box-shadow: 0 0 0 1px rgba(0, 0, 0, .1), 0 1px 3px rgba(0, 0, 0, .1);
   visibility: visible !important;
```

Comment:
Please use !important judiciously. This should strictly be the last resort

Reply:
The !important made use of here was not by me.